src/components/MainFrame/Main.tsx

```diff
@@ -1,34 +1,40 @@
 import React from "react";
 import '../../index'
 import CssBaseline from "@mui/material/CssBaseline";
-import {Checkbox, ThemeProvider} from "@mui/material";
+import {ThemeProvider} from "@mui/material";
 import {themes} from "../../themes/themes";
 import Navigation from "../Navigation/Navigation";
-import Container from "@mui/material/Container";
-import Box from "@mui/material/Box";
+import Write from "../Write/Write";
```

Comment:
/themes/themes , Navigation/Navigation - Ordnerstruktur schon ziemlich tief, verstehe die Dopplung hier nicht.

Reply:
I dont like how intelliJ displays folders and files... confuses me when files and folders are mixed